index.js

```diff
@@ -9,5 +11,30 @@ module.exports = {
     this._super.included.apply(this, arguments);
 
     forceHighlander(this.project);
+
+    let app = this._findHost();
+    if (!app.tests) {
+      let checker = new VersionChecker(this.parent).for('ember-cli-babel', 'npm');
```

Comment:
I wonder if we even need `ember-cli-version-checker` here, I would have written this like:

```js
let babelAddon = this.parent.addons.find(a => a.name === 'ember-cli-babel');
let babelVersion = babelAddon && babelAddon.pkg.version;
if (babelVersion && semver.gte(babelVersion, '7.0.0')) {

}
```

Basically, ember-cli-version-checker is checking disk but we actually have access to the version in memory.

Reply:
🤷‍♂ I'm a little out of my depth here. I modeled this after how [ember-test-selectors](https://github.com/simplabs/ember-test-selectors/blob/master/index.js#L55-L74) does it, but will do whatever you think is best.

Reply:
Ya so I think what Rob is saying is that addon discovery already loads all the addon metadata as part of the build, so we don't have to use version checker here; we can instead just use the data already loaded in memory.